### PR TITLE
Fix wrong WINDOWID with tmux

### PR DESCRIPTION
**Pull request by infokiller**

### CODE REVIEW

**Comment by infokiller** [comment]
I see this was resolved in #63 so closing

**Comment by infokiller** [comment]
Oops, closed the wrong PR. Reopening.

File: notify.plugin.zsh [hunk 1]
```diff
@@ -27,15 +29,21 @@ zstyle ':notify:*' error-sound ''
 zstyle ':notify:*' error-icon ''
 zstyle ':notify:*' disable-urgent no
 zstyle ':notify:*' activate-terminal no
+zstyle ':notify:*' always-check-active-window no
 
 unset plugin_dir
 
 # store command line and start time for later
 function zsh-notify-before-command() {
-    local window_id impl
```
Comment (marzocchi) [inline notify.plugin.zsh]: ah, this `local` is actually a leftover from my latest changes (sorry!), so `window_id` in this function will always evaluate equal to `""` below.

same for `always_check_wid`, there should be a call to 

    zstyle -s ':notify:*' always-check-active-window always_check_wid

to read the value that might have been set by the user in their `.zshrc`.

File: notify.plugin.zsh [hunk 2]
```diff
@@ -27,15 +29,21 @@ zstyle ':notify:*' error-sound ''
...
+    local window_id always_check_wid
 
     last_command="$1"
     start_time=$(date "+%s")
+
+    if [[ "$_USING_XDOTOOL" == 1 &&
+        ("$window_id" == "" || always_check_wid == "yes") ]]; then
```
Comment (marzocchi) [inline notify.plugin.zsh]: shouldn't this whole block fit better in `store-window-id`? there you would not even need to check if using the xdotool version, as that hook is run only in that version.

File: notify.plugin.zsh (hunk 1 above)
Comment (infokiller) [reply notify.plugin.zsh]: Yup, I actually did the right thing originally, but then screwed up when I merged. Fixed now.

File: notify.plugin.zsh (hunk 2 above)
Comment (infokiller) [reply notify.plugin.zsh]: Yup, moved it over there.

**Comment by marzocchi** [comment]
never mind, it shows conflict only when I choose "Rebase and merge"

**Comment by marzocchi** [comment]
thanks!

**Pull request merged**